test/test_reload.py

```diff
@@ -56,7 +55,3 @@ def test_config_load_default(self, config):
     def test_config_load_factory(self, config):
         config.load()
         assert config.loaded is True
-
-    def test_reload_run_default(self, reloader):
```

Comment:
@abidlabs you shouldn't remove this test.
That was the only way to actively test if the passed config is actually correct and the server running

Reply:
This was breaking other tests. I think b/c it wasn't closed. Let me see if I can bring it back, thanks

Reply:
Brought back test but with threading (a bit more consistent with how we use `networking.Server` in the library)